clang/test/SemaSYCL/intel-fpga-local-ast.cpp

```diff
@@ -0,0 +1,537 @@
+// RUN: %clang_cc1 -fsycl-is-device -internal-isystem %S/Inputs -sycl-std=2020 -Wno-return-type -fsyntax-only -fcxx-exceptions -ast-dump -verify -pedantic %s | FileCheck %s
```

Comment:
Is -Wno-return-type,  -verify -pedantic and -fcxx-exceptions required?

Reply:
They are not required. I have removed them.